src/Honeybee_Generator_Wind_Horizontialaxis.py

```diff
@@ -134,206 +134,212 @@ def checktheinputs(_name,_simpleOrAnalytical,_powerCoefficients,_powerControl,_r
         ghenv.Component.AddRuntimeMessage(w, "Only one template can be used at a time! Please set either TemplateLargeTurbine_ or TemplateMediumTurbine_ to true not both!")
         return -1
 
-    if _powerControl == None:
-        print "The field _powerControl must be specified!"
+    if powerControl_ == None:
+        print "The field powerControl_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _powerControl must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field powerControl_ must be specified!")
         return -1
 
-    if (_powerControl < 1) or (_powerControl > 4):
-        print "The field _powerControl must be an integer between 1 and 4!"
+    if (powerControl_ < 1) or (powerControl_ > 4):
+        print "The field powerControl_ must be an integer between 1 and 4!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _powerControl must be an integer between 1 and 4")
+        ghenv.Component.AddRuntimeMessage(w, "The field powerControl_ must be an integer between 1 and 4")
         return -1
 
-    if _rotorSpeed == None:
-        print "The field _rotorSpeed must be specified!"
+    if rotorSpeed_ == None:
+        print "The field rotorSpeed_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _rotorSpeed must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field rotorSpeed_ must be specified!")
         return -1
         
-    if _rotorDiameter == None:
-        print "The field _rotorDiameter must be specified!"
+    if rotorDiameter_ == None:
```

Comment:
@antonszilasi the same here. If you need a reference you can always check this document - check number 4: https://github.com/mostaphaRoudsari/ladybug/blob/master/resources/Ladybug%20Grahpic%20Standards.md